### ISSUE TITLE
Increase LDAP limit size

### ISSUE THREAD

**Issue by nemochovsky** [YunoHost/issues#1724]
After adding approx 400 users using the CLI, the following error occurred:

```error during LDAP search operation with: base='ou=users,dc=yunohost,dc=org', filter='(&(objectclass=person)(!(uid=root))(!(uid=nobody)))', attrs=['uid', 'cn', 'mail', 'mailuserquota', 'loginShell'] and exception {'desc': u'Size limit exceeded'}```

The size limit in all configuration files seems to be set high enough.
`/etc/ldap/ldap.conf`
`/usr/share/slapd/slapd.conf`
`/usr/share/slapd/slapd.init.ldif`
`/usr/share/yunohost/templates/slapd`

I was also trying:
`yunohost tools regen-conf slapd --force`
and restarting `slapd` multiple times.

Yunohost 4.1.5

Any insights would be appreciated!

**Comment by alexAubin** [comment]
Asked @zamentur about this a few minutes ago, he was saying he was recently able to add 500 users (using the experimental CSV import feature) without much trouble ? Not sure why that worked for him and not for you ...

**Comment by alexAubin** [comment]
Related to : https://github.com/YunoHost/issues/issues/1558

**Comment by ziima** [comment]
Seems that limit is 500 users by default.

To pass that, it is necessary to increase limit for search in `/etc/ldap/slapd.d/cn\=config.ldif` by setting:
```
olcSizeLimit: 5000
```
and increase a maximum size of a database in `/etc/ldap/slapd.d/cn=config/olcDatabase={1}mdb.ldif`:
```
# Increase DB max size to 50 MB
olcDbMaxSize: 52428800
```
since the default 10 MB only takes approximately 550 users.


It should be possible, though, to modify the code for `yunohost user` command so it behaves better, when more than 500 records are returned at once.

**Comment by squattertim** [comment]
On the other hand, LDAP limits should be modified by e.g. `ldapmodify`, not directly within the file.

**Comment by alexAubin** [comment]
Fixed in 4.3
